pkg/profiler/profiler.go

```diff
@@ -94,6 +100,12 @@ func NewCgroupProfiler(
 			tmp,
 		),
 		mtx: &sync.RWMutex{},
+		missingStacks: promauto.With(reg).NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "parca_agent_profiler_missing_stacks",
```

Comment:
```suggestion
				Name: "parca_agent_profiler_missing_stacks_total",
```

FYI https://prometheus.io/docs/practices/naming/

Reply:
e. g. https://github.com/parca-dev/parca/blob/6e21eb3fb2b1d8e330545b4cdb42e1a9ff7d2005/pkg/metastore/cache.go#L65-L121